docker/mongodb/Dockerfile

```diff
@@ -1,6 +1,10 @@
 ARG MONGODB_VERSION=4.0
 FROM mongo:${MONGODB_VERSION}
 
+RUN if [ "$MONGO_MAJOR" = 6.0 ] || [ "$MONGO_MAJOR" = 7.0 ]; then \
```

Comment:
Installing `bc` is cheap. Let's just add it unconditionally instead of keep updating this line going forward